### PR TITLE
RDoc-791 Server > Administration > Monitoring > SNMP > Zabbix

### PR DESCRIPTION
**Pull request by ml054**

### CODE REVIEW

File: Documentation/4.0/Raven.Documentation.Pages/server/administration/SNMP/setup-zabbix.markdown [hunk 1]
```diff
@@ -0,0 +1,80 @@
+﻿# Monitoring : How to setup Zabbix monitoring
+
+RavenDB 4.0 supports [SNNP](./snmp), which means that with a few quick steps you can monitor your server using Zabbix.
```
Comment (ppekrol) [inline Documentation/4.0/Raven.Documentation.Pages/server/administration/SNMP/setup-zabbix.markdown]: drop 4.0, it is hard to maintain this later and it is not needed in this context.

File: Documentation/4.0/Raven.Documentation.Pages/server/administration/SNMP/snmp.markdown [hunk 2]
```diff
@@ -0,0 +1,152 @@
+﻿# Monitoring : SNMP Support
+
+Simple Network Management Protocol (SNMP) is an Internet-standard protocol for collecting and organizing information 
+about managed devices on IP networks, and is used primarily for monitoring network services. SNMP exposes management 
+data in the form of variables (metrics) which describe the system status and configuration. These metrics can then be 
+remotely queried (and, in some circumstances, manipulated) by managing applications.
+
+In RavenDB 4.0 we have support for SNMP which allows monitoring tools like [Zabbix](https://www.zabbix.com), [PRTG](https://www.paessler.com/prtg)
```
Comment (ppekrol) [inline Documentation/4.0/Raven.Documentation.Pages/server/administration/SNMP/snmp.markdown]: drop 4.0

File: Documentation/4.0/Raven.Documentation.Pages/server/administration/SNMP/snmp.markdown [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+about managed devices on IP networks, and is used primarily for monitoring network services. SNMP exposes management 
+data in the form of variables (metrics) which describe the system status and configuration. These metrics can then be 
+remotely queried (and, in some circumstances, manipulated) by managing applications.
+
+In RavenDB 4.0 we have support for SNMP which allows monitoring tools like [Zabbix](https://www.zabbix.com), [PRTG](https://www.paessler.com/prtg) 
+and [Datadog](https://www.datadoghq.com/) direct access to the internal details of RavenDB. We expose a long list of metrics: CPU and memory usage,
+server total requests, the loaded databases... And also database specific metrics like the number of indexed items per 
+second, document writes per second, storage space each database takes and so on. 
+
+You can still monitor what is going on with RavenDB directly from the studio, or by using one of our monitoring tools,
```
Comment (ppekrol) [inline Documentation/4.0/Raven.Documentation.Pages/server/administration/SNMP/snmp.markdown]: Studio

File: Documentation/4.0/Raven.Documentation.Pages/server/administration/SNMP/snmp.markdown [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+is preferable.
+
+{NOTE:Note}
+SNMP support is available for enterprise licenses only.
+{NOTE/}
+
+
+### Enabling SNMP in RavenDB
+
+RavenDB 4.0 is already configured to support SNMP and all you have to do is enable it and restart the server.
```
Comment (ppekrol) [inline Documentation/4.0/Raven.Documentation.Pages/server/administration/SNMP/snmp.markdown]: drop 4.0

File: Documentation/4.0/Raven.Documentation.Pages/server/administration/SNMP/snmp.markdown [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+    "Monitoring.Snmp.Community": "YourString"
+    ...
+}
+{CODE-BLOCK/}
+
+The community string is used like a password. It is sent with each SNMP Get request and 
+allows or denies access to the monitored device.
+
+### The Metrics
+
```
Comment (ppekrol) [inline Documentation/4.0/Raven.Documentation.Pages/server/administration/SNMP/snmp.markdown]: missing information in this section about the endpoint /monitoring/snmp/oids

File: Documentation/4.0/Raven.Documentation.Pages/server/administration/SNMP/setup-zabbix.markdown (hunk 1 above)
Comment (ml054) [reply Documentation/4.0/Raven.Documentation.Pages/server/administration/SNMP/setup-zabbix.markdown]: done

File: Documentation/4.0/Raven.Documentation.Pages/server/administration/SNMP/snmp.markdown (hunk 2 above)
Comment (ml054) [reply Documentation/4.0/Raven.Documentation.Pages/server/administration/SNMP/snmp.markdown]: done

File: Documentation/4.0/Raven.Documentation.Pages/server/administration/SNMP/snmp.markdown (hunk 3 above)
Comment (ml054) [reply Documentation/4.0/Raven.Documentation.Pages/server/administration/SNMP/snmp.markdown]: done

File: Documentation/4.0/Raven.Documentation.Pages/server/administration/SNMP/snmp.markdown (hunk 4 above)
Comment (ml054) [reply Documentation/4.0/Raven.Documentation.Pages/server/administration/SNMP/snmp.markdown]: done

File: Documentation/4.0/Raven.Documentation.Pages/server/administration/SNMP/snmp.markdown (hunk 5 above)
Comment (ml054) [reply Documentation/4.0/Raven.Documentation.Pages/server/administration/SNMP/snmp.markdown]: done

**Comment by ml054** [comment]
Pushed my changes. @ppekrol please review.

**Pull request merged**